releasinator.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Why?

We already have fabfile. What should new contributors use to build? Etc.